Controller/PayumController.php

```diff
@@ -8,15 +8,19 @@
 
 abstract class PayumController extends AbstractController
 {
-    /**
-     * @return Payum
-     */
-    protected function getPayum()
+    protected Payum $payum;
+
+    public function __construct(Payum $payum)
```

Comment:
This is a BC break. If someone extends this controller, they would now need to pass this instance in the constructor. I think this parameter should be nullable, and if it's not passed through, we can use `$this->get('payum')` as the default value, and also trigger a deprecation notice with a message indicating that this object needs to be passed through the constructor, and it will be required in the next major version

Reply:
You are right, I will revert thesse changes and will look into this in the new Symfony 6 branch.